app/helpers/wiki_helper.rb

```diff
@@ -40,22 +40,30 @@ def wiki_page_options_for_select(pages,
                                               ids:)
   end
 
-  def breadcrumb_for_page(page, action = nil)
-    related_pages = page.ancestors.reverse
+  def breadcrumb_for_page(project, page, action = nil)
+    breadcrumbs = []
+    breadcrumbs << project_breadcrumb(project)
```

Comment:
Maybe we should add the module "Wiki" in between here. I stumbled multiple times over the fact that it is missing (see other comments I made). That would mean however, if you have a default Wiki page called "Wiki", there would be two duplicated entries.. I think we have to discuss that with the product team.